tests/unit/dataactvalidator/test_c17_award_financial.py

```diff
@@ -0,0 +1,149 @@
+                                             ussgl487200_downward_adjus_cpe = None,
+                                             ussgl488100_upward_adjustm_cpe = None,
+                                             ussgl488200_upward_adjustm_cpe = None,
+                                             ussgl490100_delivered_orde_cpe = None,
+                                             ussgl490100_delivered_orde_fyb = None,
+                                             ussgl490200_delivered_orde_cpe = None,
+                                             ussgl490800_authority_outl_cpe = None,
+                                             ussgl490800_authority_outl_fyb = None,
+                                             ussgl493100_delivered_orde_cpe = None,
+                                             ussgl497100_downward_adjus_cpe = None,
+                                             ussgl497200_downward_adjus_cpe = None,
+                                             ussgl498100_upward_adjustm_cpe = None,
+                                             ussgl498200_upward_adjustm_cpe = None)
+    award_fin_two = AwardFinancialFactory(transaction_obligated_amou = None,
+                                             ussgl480100_undelivered_or_cpe = None,
+                                             ussgl480100_undelivered_or_fyb = None,
+		                                     ussgl480200_undelivered_or_cpe = None,
+                                             ussgl480200_undelivered_or_fyb = None,
+                                             ussgl483100_undelivered_or_cpe = None,
+                                             ussgl483200_undelivered_or_cpe = None,
+                                             ussgl487100_downward_adjus_cpe = None,
+                                             ussgl487200_downward_adjus_cpe = None,
+                                             ussgl488100_upward_adjustm_cpe = None,
+                                             ussgl488200_upward_adjustm_cpe = None,
+                                             ussgl490100_delivered_orde_cpe = None,
+                                             ussgl490100_delivered_orde_fyb = None,
+                                             ussgl490200_delivered_orde_cpe = None,
+                                             ussgl490800_authority_outl_cpe = None,
+                                             ussgl490800_authority_outl_fyb = randint(1,1000),
+                                             ussgl493100_delivered_orde_cpe = None,
+                                             ussgl497100_downward_adjus_cpe = None,
+                                             ussgl497200_downward_adjus_cpe = None,
+                                             ussgl498100_upward_adjustm_cpe = None,
+                                             ussgl498200_upward_adjustm_cpe = None)
+    award_fin_three = AwardFinancialFactory(transaction_obligated_amou = None,
```

Comment:
Good point, I changed one of these cases to be 0 instead of null, the point of the multiple cases here is just to check a few different randomly chosen fields, plus one with a group of non-null fields.